content/initiatives/codegen.md

```diff
@@ -0,0 +1,47 @@
++++
+title = "Making Rust faster than C"
+slug = "codegen"
+template = "initiatives/initiative.html"
+
+[extra]
+funders = [
+    "tweedegolf"
+]
+
+blogposts = [
+]
++++
+
+### Making Rust faster than C
+
+Rust, being a systems language, should be best in class for performance-critical software. While the performance difference with C is small, it currenly blocks Rust alternatives from being adopted.
```

Comment:
```suggestion
Rust, being a systems language, should be best in class for performance-critical software. While the performance difference with C is small, it currently blocks Rust implementations from being adopted.
```

Reply:
e.g. `Zig` is een "rust alternative". misschien nog wat verder sleutelen sws want het loopt nog niet superlekker.

Reply:
maybe swap the two sentences here? i.e. lead with "rust's (lack of) performance limits adoption in the places where it matters"